server/pbench/bin/pbench-sync-satellite

```diff
@@ -283,14 +283,11 @@ echo "$TS: end - $(timestamp)" | tee -a $mail_content
 echo "$TS: duration (secs): $duration" | tee -a $mail_content
 echo "$TS: Total $nprocessed files processed, with $nfailed_md5 md5 failures and $nerrs errors" | tee -a $mail_content
 
-# if we processed any files, or there were errors, mail the report
```

Comment:
Can we fix the content here to be like the others where they have a subject line?